### PR TITLE
Adjust the pin definitions for the hardware section of the Datalogger board.

### PR DESCRIPTION
**Pull request by cold63**
*By completing this PR sufficiently, please help us to review and improve the quality of this PR and Release Notes*

-----------
## Description of Change
Please describe your proposed Pull Request and it's impact. \
- Misc:
Adjust the pin definitions

## Tests and Environments 
Please describe on what Hardware, SDK, IDE, OS that tested this Pull Request and how if necessary.  \

- Datalogger iot Board /AMB26
- ambd_arduino V3.1.7
- Arduino IDE 2.3.2
- Windows 10

### CODE REVIEW

File: Arduino_package/hardware/variants/datalogger_amb26/variant.h [hunk 1]
```diff
@@ -96,9 +96,6 @@ extern void wait_for_debug(void);
 #define A6                                      AMB_D9  // PB_3 3.3V    CH6
 
 /* LED pin mapping */
-#define LED_G                                   AMB_D3  // Green
-#define LED_R                                   AMB_D2  // Red
-#define LED_B                                   AMB_D1  // Blue
 #define LED_BUILTIN                             LED_G   // Green
```
Comment (M-ichae-l) [inline Arduino_package/hardware/variants/datalogger_amb26/variant.h:99]: I will put "NC" for "LED_BUILTIN", as there is no "LED_G" anymore.

File: Arduino_package/hardware/variants/datalogger_amb26/variant.h [hunk 2]
```diff
@@ -96,10 +96,7 @@ extern void wait_for_debug(void);
 #define A6                                      AMB_D9  // PB_3 3.3V    CH6
 
 /* LED pin mapping */
-#define LED_G                                   AMB_D3  // Green
-#define LED_R                                   AMB_D2  // Red
-#define LED_B                                   AMB_D1  // Blue
-#define LED_BUILTIN                             LED_G   // Green
```
Comment (M-ichae-l) [inline Arduino_package/hardware/variants/datalogger_amb26/variant.h:102]: I have changed LED_BUILTIN as NC for the LED_G no more exist.

**Review by M-ichae-l — APPROVED** [review]

File: Arduino_package/hardware/variants/datalogger_amb26/variant.h (hunk 2 above)
Comment (cold63) [reply Arduino_package/hardware/variants/datalogger_amb26/variant.h:102]: Please assign AMB_D3 to be used for LED_BUILTIN. It is used in my circuit. I did not notice the issue, sorry.

Comment (M-ichae-l) [reply Arduino_package/hardware/variants/datalogger_amb26/variant.h:102]: > Please assign AMB_D3 to be used for LED_BUILTIN. It is used in my circuit. I did not notice the issue, sorry.

Lets refer to the latest commit 666c2a9f111fe682f98cfb71478ab6139c81bc9f

**Pull request merged**